buildbot/master/master.cfg

```diff
@@ -462,26 +469,31 @@ def rpm(rpmbase, mockbase, mockconfig, rpmarch, mockarch):
 
     # link the new rpm into the repo
     system_map = {'centos6': '6', 'centos7': '7'}
-    rpm_package_area = RPM_REPO + "/" + system_map[rpmbase] + "/" + rpmarch + "/packages"
+    rpm_package_area = RPM_REPO / system_map[rpmbase] / + rpmarch / "packages"
```

Comment:
I know the RPM builders are untested, but wouldn't this:

    + rpmarch

be a TypeError? I don't think unary `+` works on strings or paths.

Reply:
Ah yes thank you for spotting that! It was just meant to be `/ rpmarch`, silly typo I'll fix it now :+1:

Reply:
Done :smile: